scripts/monitors/check_wsgi.py

```diff
@@ -1,43 +0,0 @@
-#!/usr/bin/python2.6
```

Comment:
Instead of modifying this shebang, took the opportunity to entirely remove the file which isn't used anymore.